src/test/java/FizzBuzzTest.java

```diff
@@ -1,85 +1,133 @@
 		final ApplicationContext context = new ClassPathXmlApplicationContext(TestConstants.SPRING_XML);
+		// set the value of the property fb of the current object declared on line 39 to the return value of the method getBean on the object context of type ApplicationContext declared on line 48 passing in the test constant STANDARD_FIZZ_BUZZ casted to the type FizzBuzz imported from import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.FizzBuzz on line 30
 		this.fb = (FizzBuzz) context.getBean(TestConstants.STANDARD_FIZZ_BUZZ);
+		// set the print stream for this class to System.out, by setting the value of the property out of the current object declared on line 37
 		this.out = System.out;
+		// call the method close on the object context declared on line 48 casted to type ConfigurableApplicationContext imported from org.springframework.context.ConfigurableApplicationContext on line 25
 		((ConfigurableApplicationContext) context).close();
 	}
 
 	/**
+	 * tearDown method on class FizzBuzzTest declared on line 35 deconstructs object and sets everything back to defaults
 	 * @return void
 	 */
 	@After
 	public void tearDown() {
+		// set the system output to the value of the property out on the current object, which should be set to System.out due to it being set on line 52 in method setUp of class FizzBuzzTest declared on line 35
 		System.setOut(this.out);
 	}
 
 	/**
+	 * do the fizzbuzz; see method testFizzBuzz on class FizzBuzzTest declared on line 35 declared on line 98
 	 * @param n int
 	 * @param s String
 	 * @throws IOException
 	 */
 	private void doFizzBuzz(final int n, final String s) throws IOException {
+		// create a constant of type ByteArrayOutputStream imported from java.io.ByteArrayOutputStream on line 10 called baos
 		final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+		// create a constant of type BufferedOutputStream imported from java.io.BufferedOutputStream on line 8 called bos, and pass in baos declared on line 75 to the constructor
 		final BufferedOutputStream bos = new BufferedOutputStream(baos);
+		// set the system output to the returned object of the constructor for PrintStream imported from java.io.PrintStream on line 14 passing in bos declared on line 77
 		System.setOut(new PrintStream(bos));
 
+		// call the method fizzBuzz on the property fb of the current object, passing in the constant parameter n of type int
 		this.fb.fizzBuzz(n);
 
+		// flush the system ouput by running the method flush on the property out on the class System
 		System.out.flush();
+		// create a string variable called platformDependentExpectedResult which replaces all of the newlines in the constant parameter s of type string with the system line seperator, fetched by passing "line.seperator" to the method getProperty on class System
 		String platformDependentExpectedResult = s.replaceAll("\\n", System.getProperty("line.separator"));
```

Comment:
Notes for anyone in the future; this line of code is currently not Enterprise Quality&trade; due to the fact that it does not import `com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.Constants` and use the constant `LINE_SEPERATOR`. To make this Enterprise Quality&trade;, is it possible for somebody to start a pull request to change this singular line of code and make it follow industry standard by adding as many imports as possible to greatly optimize the code?